tests/base/src/test/java/org/keycloak/tests/tracing/TracingProviderTest.java

```diff
@@ -372,7 +311,7 @@ public void traceMultipleAlwaysEnded() {
     }
 
     void runOnServer(TracingConsumer tracing) {
-        getTestingClient().server(TEST_REALM_NAME).run(session -> {
+        runOnServer.run(session -> {
```

Comment:
It's confusing that there is a RunOnServerClient instance and a method both named `runOnServer`. `runOnServerTracing(...)` or something may make it clearer, wdyt?

Reply:
If it's confusing to you, there's a big chance it's confusing to other people. I thought the named lambda param is sufficient (`runOnServer(tracing ->`), but renaming works to me.